client/signup/steps/plans/index.jsx

```diff
@@ -146,6 +146,66 @@ export class PlansStep extends Component {
 			);
 		}
 
+		return (
+			<ProvideExperimentData
+				name="calypso_mobile_plans_page_with_billing"
+				options={ { isEligible: isMobile() && 'onboarding' === this.props.flowName } }
+			>
+				{ ( isLoading, experimentAssignment ) => {
+					if ( isLoading ) {
+						return this.renderLoading();
+					}
+
+					// This allows us to continue with the other experiments.
+					if ( experimentAssignment?.variationName === null ) {
```

Comment:
We want to keep returning the monthly plans experiment code also on desktop, where `experimentAssignment` will be `null` as we don't fetch `"calypso_mobile_plans_page_with_billing"` experiment. So I suggest using the [same condition as in ExPlat Client code](https://github.com/Automattic/wp-calypso/blob/trunk/packages/explat-client-react-helpers/src/index.tsx#L120) that will cover all the cases except when the variation is `treatment`:

```suggestion
					if ( ! experimentAssignment?.variationName ) {
```